sass/style.css

```diff
@@ -1,5 +1,171 @@
 /* // @import 
 // @mixin */
+*,
+*:before,
+*:after {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+body{
+        font-family: "Roboto", sans-serif;
+}
 :root {
-    background : white;
-}
\ No newline at end of file
+    --white: #fff;
+    --smooth: all .3s ease-in;
+    --ticket: #ffdd7b;
+    --blue: #283869;
+    --pink: #ff6adf;
+    --green: #92fa3c;
+    --purple: #737d9e;
+}
+
+img {
+    max-width: 100%;
+    width: 100%;
+}
+
+.container {
+    max-width: 1170px;
+    width: 100%;
+    margin: 0 auto;
+    padding: 0 10px;
+} 
+
+/* header */
+header {
+    position: absolute;
+    width: 100%;
+}
+.navbar {
+    display: flex;
+    padding: 30px;
+    justify-content: space-between;
+    align-items: center;
+}
+/* logo */
+.navbar__logo img{
+    margin-left: 137px;
```

Comment:
margin-left? img not vertically centered?

![image](https://user-images.githubusercontent.com/47839903/205648885-7b50a363-a2f2-4452-bce7-dbbe48a96ca0.png)